src/seedu/addressbook/common/Utils.java

```diff
@@ -34,4 +36,16 @@ public static boolean elementsAreUnique(Collection<?> items) {
         }
         return true;
     }
+
+
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    String getPrintableString(Printable... printables){
+        String output = "";
+
+        for(Printable p : printables) String.join(" ", output, p.getPrintableString());
```

Comment:
Try to follow the correct formatting of for loop as specified in the coding standard.